2023/assignments/bibek/bibek.md

```diff
@@ -0,0 +1,144 @@
+# <p align='center'>Git Workshop 2023 <p>
+
+### <p align='center'>`Bibek Dhungana`</p>
```

Comment:
@th3-roGue-deV1l, remove backticks (`) from this line.

Reply:
done